README.md

```diff
@@ -9,7 +9,7 @@ An admin tool that allows you to control a block layout across units.
 | Moodle Version   | Branch            |
 |------------------|-------------------|
 | Moodle 3.9 - 4.1 | master            |
-| Moodle 4.4       | MOODLE_404_STABLE |
+| Moodle 4.4 - 4.5 | MOODLE_404_STABLE |
```

Comment:
Please add 4.5 support here https://github.com/catalyst/moodle_tool_blocksmanager/blob/MOODLE_404_STABLE/version.php#L31 

Can  you please also bump version please?